### PR TITLE
Object Manipulation 1 | Yang An Boada

### PR DESCRIPTION
**Pull request by yanganboada**
Screen Shot 2020-10-05 at 6 37 38 PM

### CODE REVIEW

File: object-manipulation-1/src/get-keys.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/* eslint-disable no-unused-vars */
+function getKeys(object) {
+  const keys = Object.keys(object);
+  return keys;
+}
```
Comment (thebearingedge) [inline object-manipulation-1/src/get-keys.js:5]: Please do not use `Object.keys` for this

File: object-manipulation-1/src/get-values.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/* eslint-disable no-unused-vars */
+function getValues(object) {
+  const values = Object.values(object);
```
Comment (thebearingedge) [inline object-manipulation-1/src/get-values.js:3]: Please do not use `Object.values` for this.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: object-manipulation-1/src/get-keys.js (hunk 1 above)
Comment (yanganboada) [reply object-manipulation-1/src/get-keys.js:5]: Sorry for missed instruction on this, I will revising it now.

File: object-manipulation-1/src/get-values.js (hunk 2 above)
Comment (yanganboada) [reply object-manipulation-1/src/get-values.js:3]: Sorry for missed instruction on this, I will revising it now.

Comment (yanganboada) [reply object-manipulation-1/src/get-values.js]: Issue resolved with new commit 
ab67dc5

File: object-manipulation-1/src/get-keys.js (hunk 1 above)
Comment (yanganboada) [reply object-manipulation-1/src/get-keys.js]: Issue resolved with new commit 
ab67dc5

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**